passKit/Helpers/DefaultsKeys.swift

```diff
@@ -11,20 +11,16 @@ import SwiftyUserDefaults
 
 public var Defaults = DefaultsAdapter(defaults: UserDefaults(suiteName: Globals.groupIdentifier)!, keyStore: DefaultsKeys())
 
-public enum PGPKeySource: String, DefaultsSerializable {
-    case url, armor, files, itunes
+public enum KeySource: String, DefaultsSerializable {
+    case url, armor, file, itunes
 }
 
 public enum GitAuthenticationMethod: String, DefaultsSerializable {
     case password, key
 }
 
-public enum GitSSHKeySource: String, DefaultsSerializable {
-    case file, armor, url
-}
-
 public extension DefaultsKeys {
-    var pgpKeySource: DefaultsKey<PGPKeySource?> { .init("pgpKeySource") }
+    var pgpKeySource: DefaultsKey<KeySource?> { .init("pgpKeySource") }
```

Comment:
I noticed one problem in the current TestFlight version with the changes in this file: The previous `file` is now `itunes` for both PGP and SSH keys. So, if a user had imported their keys via iTunes before the update, it will now be shown as imported from files. As far as I can see, this information is only relevant for the "✓" displayed in the import dialog to flag the used method. Shall we keep it since it is only UI related or shall we introduce another enum case for the import from files and keep `file` for iTunes, @mssun?

Reply:
Do you mean this UI issue is only valid for people using iTunes upgrading from older version? I think it's okay if there is no other side effect.

Reply:
Yes, `url` and `armor` have remained untouched. `itunes` is a new case and thus it is not known to `UserDefaults`. Only `file` changed to `itunes`, which causes the misinterpretation if iTunes was used for the key import. There are no errors, though, as no cases have been removed or renamed.